tests/Issues/IssueOrIfExplicitBoolCompare/Fixture/fixture.php.inc

```diff
@@ -28,7 +28,7 @@ class Fixture
      */
     public function run($a = '"', $b)
     {
-        if (! $a) {
+        if ($a === '' || $a === '0') {
```

Comment:
This also fix combination between `ExplicitBoolCompareRector` + `ChangeOrIfReturnToEarlyReturnRector` which previously got the bool compare one got skipped and require double run 🎉🎉🎉